docs/guides/kubeflow/running-kubeflow-pipelines.md

```diff
@@ -5,4 +5,79 @@ sidebar_position: 2
 
 # Kubeflow Pipelines
 
-Kubeflow Pipelines is a platform for building and deploying portable, scalable machine learning (ML) workflows based on Docker containers. It is a part of the Kubeflow project, which aims to make running ML workloads on Kubernetes simple, portable and scalable.
\ No newline at end of file
+Kubeflow Pipelines is a platform for building and deploying portable, scalable machine learning (ML) workflows based on Docker containers. It is a part of the Kubeflow project, which aims to make running ML workloads on Kubernetes simple, portable and scalable. This guide aims to explain and demonstrate how to run a Kubeflow Pipeline.
+
+### What is a Pipeline?
```

Comment:
Might want to add a line at the bottom of this section explaining that Pipelines are only run through Experiments. That wasn't obvious to me when I was first putting together the intro content, and the Kubeflow documentation isn't very clear about it, either.